src/content/governance/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Ethereum Governance
+description: An introduction to how decisions about Ethereum are made.
+lang: en
+sidebar: true
+---
+
+# Introduction to Ethereum Governance {#introduction}
+
+_If no one owns Ethereum, how are decisions about past and future changes to Ethereum made? Ethereum Governance refers to the process that allows such decisions to be made_
+
+## What is Governance? {#what-is-governance}
+
+The Cambridge dictionary defines governance as:
+
+> _the way that organizations or countries are managed at the highest level, and the systems for doing this_
+
+In essence, it is the systems in place that allows decisions to be made. In a typical organizational structure, the directors' board has the final say in decision-making. In a political system, elected officials advocate for bills that best represent their constituent's desires.
+
+## Decentralized Governance {#decentalized-governance}
+
+No one person owns the Ethereum protocol, but decisions still need to be made about implementing changes best to ensure the longevity and prosperity of the network. This lack of ownership makes traditional organizational governance an incompatible solution.
+
+## Ethereum Governance {#ethereum-governance}
+
+Ethereum Governance is the process by which protocol changes are made. It's important to point out that this process isn't related to how people and applications use the protocol - Ethereum is permissionless. Anyone from anywhere in the world can participate in on-chain activities. There are no rules set for who can or cannot build an application or send a transaction. However, there is a process to propose changes to the core protocol, which these applications run on top of. Since so many people depend on Ethereum's stability, there is a very high coordination threshold for core changes, including social and technical processes, to ensure any changes to Ethereum are secure and acceptable to the community
```

Comment:
```suggestion
Ethereum Governance is the process by which protocol changes are made. It's important to point out that this process isn't related to how people and applications use the protocol - Ethereum is permissionless. Anyone from anywhere in the world can participate in on-chain activities. There are no rules set for who can or cannot build an application or send a transaction. However, there is a process to propose changes to the core protocol, which these applications run on top of. Since so many people depend on Ethereum's stability, there is a very high coordination threshold for core changes, including social and technical processes, to ensure any changes to Ethereum are secure and widely supported by the community
```